cypress/integration/test-helper.js

```diff
@@ -0,0 +1,13 @@
+export const testRequestResponse = (uri, responseCode) => {
+	cy.request(uri).then((response) => {
+         expect(response.status).to.eq(responseCode)
```

Comment:
Could you tidy up the unneeded lines (4-6 and 9-12). Also check the spacing of lines 2 and 3. Should be 4 lines or a tab between each level of code indenting :)